src/base/projects/Header.svelte

```diff
@@ -222,63 +202,49 @@
 </style>
 
 <header>
-  <div class="commit">
-    {#if commit === project.head}
-      <div class="branch">
-        {project.meta.defaultBranch}
-      </div>
-      <div class="hash">
-        {utils.formatCommit(commit)}
-      </div>
-    {:else}
-      <div class="hash desktop">
-        {commit}
-      </div>
-      <div class="hash mobile">
-        {utils.formatCommit(commit)}
-      </div>
-    {/if}
-  </div>
-  <div class="anchor">
-    {#if anchors}
-      {#await getAnchor}
-        <Loading small margins />
-      {:then anchor}
-        {#if anchor}
-          <!-- commit is head and latest anchor  -->
-          {#if commit == anchor[0] && commit === project.head}
-            <span class="anchor-widget anchor-latest">
-              <span class="anchor-label" title="{anchors}">latest 🔐</span>
-            </span>
-          <!-- commit is not head but latest anchor  -->
-          {:else if commit == anchor[0] && commit !== project.head}
-            <span class="anchor-widget" on:click={() => commit = project.head}>
-              <span class="anchor-label" title="{anchors}">latest 🔐</span>
-            </span>
-          <!-- commit is not head a stale anchor  -->
-          {:else if anchor?.includes(commit)}
-            <span class="anchor-widget" on:click={() => commit = anchor[0]}>
-              <span class="anchor-label" title="{anchors}">stale 🔒</span>
-            </span>
-          <!-- commit is not anchored, could be head or any other commit  -->
+  {#if commit}
```

Comment:
The if clause here allows us to have initially an undefined commit, and after mounting the nested routes render this part.